spec/mysql2/client_spec.rb

```diff
@@ -1078,19 +1078,24 @@ def run_gc
     end
 
     it "returns multiple session track type values when available" do
-      @client.query("SET @@SESSION.session_track_transaction_info='CHARACTERISTICS'")
+      @client.query("SET @@SESSION.session_track_transaction_info='CHARACTERISTICS';")
```

Comment:
```suggestion
      @client.query("SET @@SESSION.session_track_transaction_info='CHARACTERISTICS'")
```

This semicolon shouldn't be needed.